app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -27,7 +28,15 @@ public static EventListFragment newInstance(String plusId) {
         fragment.setArguments(arguments);
         return fragment;
     }
-    
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        final ViewGroup.LayoutParams layoutParams = getListView().getLayoutParams();
+        layoutParams.width = (int) getResources().getDimension(R.dimen.responsive_width);
```

Comment:
Yes, it crashes. It was working when I do it with styles. Maybe not programatically. I will look into it right now.